module.json

```diff
@@ -24,29 +24,13 @@
     }
   ],
   "dependencies": {
-    "ble": "^2.0.0"
+    "ble": "^2.0.0",
+    "nrf51-sdk": "*"
```

Comment:
would be nice to say which version of the nrf51-sdk we depended on. ^8.0.0 perhaps.